app/config/tstvhq/events.json

```diff
@@ -1,98 +1,73 @@
 {
     "events": [{
-            "name": "Blood Libel: Satanic Panic, QAnon, and Anti-Semitism",
-            "description": "Evan Anderson presents the anti-Semitic roots of Satanic Panic and QAnon.",
-            "room": "ritualChamber",
-            "start": "2020-10-17T20:00:00-04:00",
-            "end": "2020-10-17T21:00:00-04:00",
+            "name": "MENTAL HEALTH ABUSE & MALPRACTICE",
+            "description": "Grey Faction with Lucien Greaves. On the abuse and malpractice in the mental health industry.",
```

Comment:
Style nit: I think it reads better without the "the" ("On abuse and malpractice in the mental health industry")